portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,12 +69,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
-    String userComment = getParameter(request, "text-input", "not-working");
+    String userComment = getParameter(request, "text-input", "empty");
+
+    if (userComment == "empty"){
```

Comment:
This works because Strings are interned, but I think you should generally stick with .equals for object comparisons.